test/base/AppWithDefaultMockFixtures.scala

```diff
@@ -37,12 +36,9 @@ import scala.concurrent.{ExecutionContext, Future}
 trait AppWithDefaultMockFixtures extends BeforeAndAfterEach with GuiceOneAppPerSuite with GuiceFakeApplicationFactory with MockitoSugar {
   self: TestSuite =>
 
-  override def beforeEach(): Unit =
-    Mockito.reset(
-      mockSessionRepository,
-      mockDataRetrievalActionProvider,
-      mockCheckArrivalStatusProvider
-    )
+  override def beforeEach(): Unit = {
+    reset(mockSessionRepository); reset(mockDataRetrievalActionProvider); reset(mockCheckArrivalStatusProvider)
```

Comment:
I think this can be written like 

reset(mockSessionRepository, mockDataRetrievalActionProvider, mockCheckArrivalStatusProvider)

Reply:
@ajchilts I thought so too 🤔 But the following warning gets thrown
<img width="699" alt="Screenshot 2022-10-24 at 11 31 05" src="https://user-images.githubusercontent.com/43777106/197506511-b7285c56-2e65-48f9-8786-bf086cb8ec0b.png">

Reply:
And I'm hesitant to silence that particular warning incase we have a genuine case of it elsewhere 😄

Reply:
Let's stick with you implementation then. :-)